src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int firstSide;
+    private final int secondSide;
+    private final int height;
+
+    public IsoscelesTrapezoid(Colors color, int firstSide, int secondSide, int height) {
+        setColor(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.height = height;
+
```

Comment:
[Please don’t add redundant empty lines to your code.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#please-dont-add-redundant-empty-lines-to-your-code)